src/packages/excalidraw/index.tsx

```diff
@@ -236,3 +236,5 @@ export {
 } from "../../utils";
 
 export { Sidebar } from "../../components/Sidebar/Sidebar";
+
+export * from "../../actions";
```

Comment:
We can have a more granular control over actions we want to expose, let me know if some of the actions need to be blacklisted.